packages/mattermost-plugin/prod.webpack.config.js

```diff
@@ -57,17 +58,17 @@ const clientTransformRules = (pluginRoot) => {
 module.exports = (config) => {
   const minimize = config.minimize === 'true'
   return {
-    entry: "./index",
-    mode: "development",
-    devtool: "source-map",
+    entry: path.join(PLUGIN_ROOT, './index'),
```

Comment:
hey @Dschoordsch could you make sure i didn't bork anything here? i used absolute paths instead of relative so we can call it with `pnpm webpack` because `pnpm --filter parabol-mattermost-plugin webpack` tries to run the `webpack` script.